src/components/SideBar/SideBarComponents/searchComponet.js

```diff
@@ -29,6 +29,7 @@ function localSearchCOmponent({ classes }) {
   const [options, setOptions] = React.useState([]);
   const loading = open && options.length === 0;
   const [value, setValue] = React.useState([]);
+  const [valueReverse, setValueReverse] = React.useState([]);
```

Comment:
Can we use the same value in for this we don't need a separate state for this. While pushing a new value to setValue you can trigger that

Reply:
I tried, we couldn't. For example, the first search result array is [1], then we reverse it the search result array is still [1]. Then we get the second search result array [1,2]. After reversing it, the search result array is [2,1]. As you can see, right now, the search result value is [2,1] because we reverse it. So, when we add a new search result 3 to the search result array, the search result array will be like [2,1,3], and after we reverse it, the search result is [3,1,2].

Reply:
We are having the second state because we want the search result to reverse correctly.

Reply:
Sorry, my mistakes. I can do it. Let me update my code.

Reply:
I am making it too complicated